oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/InfluxClient.java

```diff
@@ -0,0 +1,210 @@
+     */
+    public List<QueryResult.Result> query(Query query) throws IOException {
+        if (log.isDebugEnabled()) {
+            log.debug("SQL Statement: {}", query.getCommand());
+        }
+
+        try {
+            QueryResult result = getInflux().query(query);
+            if (result.hasError()) {
+                throw new IOException(result.getError());
+            }
+            return result.getResults();
+        } catch (Exception e) {
+            throw new IOException(e.getMessage() + System.lineSeparator() + "SQL Statement: " + query.getCommand(), e);
+        }
+    }
+
+    /**
+     * Execute a query against InfluxDB with a single statement.
+     *
+     * @param query Query
+     * @return a set of {@link QueryResult.Series}s
+     * @throws IOException if there is an error on the InfluxDB server or communication error
+     */
+    public List<QueryResult.Series> queryForSeries(Query query) throws IOException {
+        List<QueryResult.Result> results = query(query);
+
+        if (CollectionUtils.isEmpty(results)) {
+            return null;
+        }
+        return results.get(0).getSeries();
+    }
+
+    /**
+     * Execute a query against InfluxDB with a single statement but return a single {@link QueryResult.Series}.
+     *
+     * @param query Query
+     * @return {@link QueryResult.Series}
+     * @throws IOException if there is an error on the InfluxDB server or communication error
+     */
+    public QueryResult.Series queryForSingleSeries(Query query) throws IOException {
+        List<QueryResult.Series> series = queryForSeries(query);
+        if (CollectionUtils.isEmpty(series)) {
+            return null;
+        }
+        return series.get(0);
+    }
+
+    /**
+     * Data management, to drop a time-series by measurement and time-series name specified. If an exception isn't
+     * thrown, it means execution success. Notice, drop series don't support to drop series by range
+     *
+     * @param measurement String
+     * @param timeBucket  long
+     * @throws IOException if there is an error on the InfluxDB server or communication error
+     */
+    public void dropSeries(String measurement, long timeBucket) throws IOException {
+        Query query = new Query("DROP SERIES FROM " + measurement + " WHERE time_bucket='" + timeBucket + "'");
+        QueryResult result = getInflux().query(query);
+
+        if (result.hasError()) {
+            throw new IOException("Statement: " + query.getCommand() + ", ErrorMsg: " + result.getError());
+        }
+    }
+
+    public void deleteByQuery(String measurement, long timestamp) throws IOException {
+        this.query(new Query("delete from " + measurement + " where time < " + timestamp + "ms"));
+    }
+
+    /**
+     * Write a {@link Point} into InfluxDB. Note that, the {@link Point} is written into buffer of InfluxDB Client and
+     * wait for buffer flushing.
+     *
+     * @param point Point
```

Comment:
From here, all comments about @param and @return are either empty and meaningless. Could you fix this?

Reply:
@dmsolr you should put the **description** after the parameter name and return name, **NOT** the type of the parameter and return object, they're meaningless, and will cheat the JavaDoc compiler, if they are already self-documented, simply remove the JavaDoc.